### PR TITLE
052020

### PR DESCRIPTION
**Pull request by antgonza**
Depends on (please review/merge those one first):
- [x] https://github.com/biocore/qiita/pull/2994
- [x] https://github.com/biocore/qiita/pull/2989

### CODE REVIEW

**Comment by antgonza** [comment]
@ElDeveloper, this is ready for review. BTW It's also deployed in qiita-rc.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Qiita changelog
 
+Version 052020
+--------------
+
+* Added Metatrascriptomicas as a data type, added a Ribosomal read filtering step and documentation on how to use it in the processing recommendations
+* Fixed issue that prevented creating new artifacts when it was the children of a public parent
+* Qiita now keeps track of the delete artifact jobs which avoids creating duplicated delete jobs and the user gets this feedback in the GUI
```
Comment (ElDeveloper) [inline CHANGELOG.md:8]:
```suggestion
* Qiita now keeps track of artifact deletion jobs, prevents submitting duplicated deletions, and the GUI is updated when an artifact is being deleted
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+* We now display the `redbiom` DB release date in the redbiom Qiita page
+* Fixed EBA-ENA duplicated sample submission in multiple preparations - this could happen when a sample existed in more than one preparation
+* Now is possible to deprecate a full preparation; this is useful when there is an unsuccessful run/preparation
```
Comment (ElDeveloper) [inline CHANGELOG.md:11]:
```suggestion
* Add the ability to deprecate a preparation; this is useful when there is an unsuccessful run or preparation
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+* The study page now has a markdown `Notes` section so users add problematic samples, explaining certain metadata columns, etc
+* Added user documentation to better explain how to split your samples in preparations
+* Removed unused-legacy code to deal with duplicated sample names while building analyses
```
Comment (ElDeveloper) [inline CHANGELOG.md:14]:
```suggestion
* Fix a bug where repeated sample names were incorrectly handled during meta-analysis (#2978). Removed unused-legacy code to deal with duplicated sample names while building analyses.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,23 @@
...
+* Improved headings in the stats page to avoid confusions
+* Fixed annoying issue that only deleted selected samples within the page viewing frame
```
Comment (ElDeveloper) [inline CHANGELOG.md:16]:
```suggestion
* Fixed issue that only deleted selected samples within the page's viewing frame
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,23 @@
...
+* Qiita now uses `gnu=True` in the `humanize.naturalsize` so the size display matches what a user sees when they `ls` the file
+* Updated code so it works with the latest version of `pandas`
```
Comment (ElDeveloper) [inline CHANGELOG.md:18]: Would you mind listing the exact version?

**Comment by ElDeveloper** [comment]
Thanks @antgonza !

**Pull request merged**